File: src/Scheduler.Data/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Scheduler.Data
{
    public class Employee      
    {
        public int EmployeeId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string EmployeeNumber { get; set; }
        public string PhoneNumber { get; set; }

        public Organization Organization { get; set; }
    }
}

File: src/Scheduler.Web/wwwroot/app/models/employee.ts
import { Position }             from '../models/schedule';

export class EmployeeList {
    availablePositions: Position[];
    employees: Employee[];
}

export class Employee {
    employeeId: number;
    firstName: string;
    lastName: string;
    employeeNumber: string;
    phoneNumber: string;
    isActive: boolean;

    employeePositionIds: number[];
}


export class CreateEmployeeAccess {
    phoneNumber: string;
    employeeId: number;
    password: string;
}
